packages/lib-classifier/src/plugins/drawingTools/components/Mark/Mark.js

```diff
@@ -110,7 +110,11 @@ const Mark = forwardRef(function Mark(
       case 'Enter': {
         event.preventDefault()
         event.stopPropagation()
-        openSubTaskPopup()
+        try {
+          openSubTaskPopup()
+        } catch (err) {
+          // this causes freehandLine to blow up when deleting a mark
+        }
```

Comment:
Do you know what the bug is in `FreehandLine` that's breaking the popup here?

Silently swallowing errors means we'll miss any bugs that arise in transcription popups. Better to open an issue for this, and fix it in the `FreehandLine` code, I think.

Reply:
I'm not seeing an error here when I draw a freehand line, then delete it. Am I missing something?